_pages/stormrider41.yaml

```diff
@@ -0,0 +1,5 @@
+---
+  githubHandle: stormrider41
+  pageURL: stormrider41.github.io/stormrider41/
```

Comment:
There are some extra spaces before each word on lines 2, 3, and 4. Make sure that those are gone so the tests will pass. 😄